locales/de-DE.json

```
@@ -1,9 +1,17 @@
 {
+  "a11y": {
+    "loading_page": "Seite wird geladen, bitte warten",
+    "loading_titled_page": "Seite {0} wird geladen, bitte warten",
+    "locale_changed": "Sprache geändert zu {0}",
+    "locale_changing": "Sprache ändern, bitte warten",
+    "route_loaded": "Seite {0} geladen"
+  },
   "account": {
     "avatar_description": "{0}'s Avatar",
     "blocked_by": "Du wurdest von diesem Account geblockt",
     "blocked_domains": "Gesperrte Domänen",
     "blocked_users": "Gesperrte Accounts",
+    "blocking": "verstopft",
```

This is not right!

We need to see the context, maybe

```suggestion
    "blocking": "blockierend",
```